package.json

```diff
@@ -54,13 +54,11 @@
     "prettier": "^1.19.1"
   },
   "scripts": {
-    "test": "npm run prepare && npm run test:eslint && npm run test:flow && npm run test:mocha",
```

Comment:
Please,  keep the `test` script. It is used by developers and on TravisCI to test if everything works correctly. 😄